Dockerfile

```
@@ -18,25 +25,19 @@ RUN dnf -y install \
   gcc \
   clang \
   python3-devel \
-  && dnf clean all \
-  && rm -rf /var/cache/dnf \
-  && rm -rf /var/lib/dnf \
-  && truncate -s0 /var/log/*.log && rm -rf /var/cache/yum
+  && microdnf clean all
+
+  RUN pip install uv
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)